pkg/render/apiserver.go

```diff
@@ -1068,8 +1070,10 @@ func (c *apiServerComponent) apiServerDeployment() *appsv1.Deployment {
 					Tolerations:        c.tolerations(),
 					ImagePullSecrets:   secret.GetReferenceList(c.cfg.PullSecrets),
 					InitContainers:     initContainers,
-					Containers:         containers,
-					Volumes:            c.apiServerVolumes(),
+					Containers: []corev1.Container{
```

Comment:
I think this needs to go back to `Containers: containers,` or else there are more changes needed here to account for the l7AdmissionControllerContainer that is conditionally added to the containers above.